### PR TITLE
Build building and buy land

### PR DESCRIPTION
**Pull request by nradenovic93**
Loads actions available to stakeholders, loads all functions, plus basic build function and buy land function.

### CODE REVIEW

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/api/Environment.java [hunk 1]
```diff
@@ -71,11 +80,47 @@ public void run() {
...
+    for (int i = 0; i < actionList.length(); i++) {
+      JSONObject action = actionList.getJSONObject(i).getJSONObject("ActionMenu");
+      JSONArray functions = action.getJSONArray("functionTypes");
+      JSONObject stakeholders = action.getJSONObject("activeForStakeholder");
+      
+      JSONArray keys = stakeholders.names();
+      Map<Integer, JSONArray> functionsMap = new HashMap<Integer, JSONArray>();
+      for (int j = 0; j < keys.length(); j++) {
+        if (stakeholders.getBoolean(keys.getString(j))) {
+          functionsMap.put(Integer.parseInt(keys.getString(j)), functions);
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/api/Environment.java]: ParseInt of a getString should be changed to `keys.getInt(j)`.

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/api/Environment.java [hunk 2]
```diff
@@ -168,15 +229,51 @@ public BuildingList loadBuildings() {
...
+  
+  /**
+   * Buys a piece of land.
+   * @param buyerId The stakeholder id of the buyer.
+   * @param polygons The polygons describing the land.
+   * @param cost The amount of money per unit of land.
+   */
+  public void buyLand(int buyerId, String polygons, int cost) {
+    logger.debug("Buying land");
+    JSONArray parameters = new JSONArray();
```
Comment (devbridie) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/api/Environment.java]: Consider refactoring this JSONArray into a dedicated object, see `Session.CloseSessionRequest`.

**Comment by devbridie** [comment]
See my comments. Besides this, looking good!

**Comment by HarmGriffioen** [comment]
:+1:

**Pull request closed without merging**